test_scripts/WebEngine/WebEngineProjection/008_Registration_appHMIType_is_array.lua

```diff
@@ -0,0 +1,42 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0273-webengine-projection-mode.md
+--
+-- Description:
+-- Check that it is possible to register App with array of HMI types [WEB_VIEW, Media].
+--
+-- Precondition:
+-- 1. SDL and HMI are started
+-- 2. WebEngine device is connected to SDL
+-- 3. WEB_VIEW and Media appHMITypes are allowed by policy
```

Comment:
@YarikMamykin 
```suggestion
-- 3. WEB_VIEW and Media appHMITypes are allowed by policy for application (App)
```

Reply:
@aderiabin Done in https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/30/commits/49885c343b74f7c2c2702a4e516748cfe9165450#diff-94382dcebc000bb68932a1e17bffe69cR10